### PR TITLE
Add implicit Int enhancing class

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: src/main/scala/lenthall/numeric/IntegerUtil.scala [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package lenthall.numeric
+
+object IntegerUtil {
+
+  private val OrdinalSuffixes = List("th", "st", "nd", "rd", "th", "th", "th", "th", "th", "th")
```
Comment (mcovarr) [inline src/main/scala/lenthall/numeric/IntegerUtil.scala]: Maybe a bit DRYer?

```scala
def ordinal(int: Int): String = {
  int + (int match {
    case 1 => "st"
    case 2 => "nd"
    case 3 => "rd"
    case _ => "th"
  })
}
```

File: src/main/scala/lenthall/numeric/IntegerUtil.scala [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package lenthall.numeric
+
+object IntegerUtil {
+
```
Comment (geoffjentry) [inline src/main/scala/lenthall/numeric/IntegerUtil.scala]: Superfluous \n

File: src/main/scala/lenthall/numeric/IntegerUtil.scala [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  private val OrdinalSuffixes = List("th", "st", "nd", "rd", "th", "th", "th", "th", "th", "th")
+
+  implicit class IntEnhanced(val value: Int) extends AnyVal {
+    def toOrdinal: String = value match {
+      case v if v.isBetweenInclusive(10, 20) => s"${v}th"
+      case v => s"$v${OrdinalSuffixes(v % 10)}"
```
Comment (geoffjentry) [inline src/main/scala/lenthall/numeric/IntegerUtil.scala]: I'd prefer to make this 2 lines - one doing the `%` and one making the string

**Comment by geoffjentry** [comment]
Modulo (pun intended) the small comments :+1:

**Comment by mcovarr** [comment]
:+1: as well delta that one nit

File: src/main/scala/lenthall/numeric/IntegerUtil.scala (hunk 1 above)
Comment (Horneth) [inline src/main/scala/lenthall/numeric/IntegerUtil.scala]: Would this replace only the list or the entire `toOrdinal` method ?
If the former I think `int +` shouldn't be there and if the latter it wouldn't work for 21st, 22nd etc..

Comment (mcovarr) [inline src/main/scala/lenthall/numeric/IntegerUtil.scala]: sure, however you think this would integrate best.

**Pull request merged**